README.md

```diff
@@ -34,14 +29,25 @@ This card is available in [HACS](https://hacs.xyz/) (Home Assistant Community St
   $ wget https://github.com/kalkih/mini-graph-card/releases/download/v0.9.4/mini-graph-card-bundle.js
   ```
 
-3. Add a reference to `mini-graph-card-bundle.js` inside your `ui-lovelace.yaml`:
+3. Add ressource as decribed below.
+
+### Add ressource
+
+If you configure Lovelace via YAML, add a reference to `mini-graph-card-bundle.js` inside your `ui-lovelace.yaml` or at the top of the *raw config editor UI*:
```

Comment:
This is actually also outdated, should now be `configuration.yaml` instead of `ui-lovelace.yaml`, also I think you could remove the `or at the top of the *raw config editor UI*` part